src/stockwatch/use_cases/importing.py

```diff
@@ -72,13 +72,30 @@ def _process_sell_transaction_row(
     isin: IsinStr,
     row: dict[str, str],
     exchanges: list[CurrencyExchange],
+    cash_settlements: list[CashSettlement],
 ) -> ShareTransaction:
     descr = row["Omschrijving"].split()
     key_index = descr.index("Verkoop")
     nr_stocks = float(descr[key_index + 1].replace(",", "."))
     curr = row["Mutatie"]
     price = Amount(float(descr[key_index + 3].replace(",", ".")), curr)
-    amount = apply_exchange(transaction_datetime, nr_stocks * price, exchanges)
+    amount = Amount(0.0)
+    if "DELISTING:" in descr:
```

Comment:
I'm usually a fan of keeping the common case first, especially as it is much shorter as this if. So it would become:
```
if "DELISTING:" not in descr:
    amount = apply_exchange(transaction_datetime, nr_stocks * price, exchanges)
else:
    # process delistings:
    ....
```

pretty minor, and it's a bit personal style I guess.

Reply:
moved this out entirely